Toolset/resources/supporting_files/property_definitions/propertyInfo.tsv

```diff
@@ -86,6 +86,7 @@ hotspots	Hotspots	Quicktime	com.livecode.pi.string	true	true
 hoverIcon	Hover icon	Icons	com.livecode.pi.imageid	true	false		0	
 hScrollbar	Horizontal scrollbar	Basic	com.livecode.pi.boolean	true	false		false		
 icon	Icon	Icons	com.livecode.pi.imageid	true	false		0	
+iconGravity	Gravity	Icons	com.livecode.pi.enum	true	false			,left,top,right,bottom,topLeft,topRight,bottomLeft,bottomRight,center,resize
```

Comment:
@livecodeali it would be nice to allow the word `empty` here or `(empty)` or something. I'm not sure how deep or risky that change might be.

Reply:
Hmm, yeah... I'm not sure either!